From 69b3f1b6098eb7ba916c4155441ecd7e0aa77fa3 Mon Sep 17 00:00:00 2001
From: Sean Mackrory <sean@cloudera.com>
Date: Fri, 7 Sep 2012 15:26:42 -0700
Subject: [PATCH 136/795] MR1: CDH-6973: Mockito needs to be distributed because hadoop-test.jar needs it at runtime
 (cherry picked from commit 4995d1349527b0a5763a85a0bc4c8a0fce0c148e)

(cherry picked from commit d008edfc0e3af52e99bf576d538a40bf344cd952)
---
 ivy.xml |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/ivy.xml b/ivy.xml
index 181350b..64e4773 100644
--- a/ivy.xml
+++ b/ivy.xml
@@ -117,7 +117,7 @@
    <dependency org="junit" name="junit" rev="${junit.version}"
                conf="test->default"/>
    <dependency org="org.mockito" name="mockito-all" rev="${mockito-all.version}" 
-               conf="test->default"/>
+               conf="common->default"/>
    <dependency org="org.vafer" name="jdeb" rev="${jdeb.version}" conf="package->master"/>
    <dependency org="org.mortbay.jetty" name="jetty-servlet-tester" rev="${jetty.version}"
                conf="test->default"/>
-- 
1.7.0.4

